lib/cn/cn_metrics.h

```diff
@@ -373,6 +373,36 @@ cn_samp_sub(
     lhs->l_vgarb -= rhs->l_vgarb;
 }
 
+static inline uint
+cn_samp_est(struct cn_samp_stats *s, uint scale)
```

Comment:
write out the rest of `estimate`!

Reply:
Apparently that's the way @alexttx wanted it, who am I to judge? :-)